smol/moca/sampler/container.py

```diff
@@ -126,6 +140,29 @@ def sampling_efficiency(self, discard=0, flat=True):
             efficiency = efficiency.mean()
         return efficiency
 
+    def get_sampled_structures(self, indices=-1):
+        """Get sampled structures for MC steps given by indices.
+
+        Args:
+            indices (list of int or int):
+                A single index or list of indices to obtain sampled structures
```

Comment:
Please explain in this docstring what the default indices=-1 will do.

Reply:
Also why not have argument "flatten" in this function as well?

Reply:
edited the code to include a `flat` keyword argument, and explained behavior in docstring.